utils/spacewalk-utils.changes

```diff
@@ -1,3 +1,5 @@
+- Add SLES 15 SP5 client tools repositories to spacewalk-common-channels.ini
```

Comment:
```suggestion
- Add SLES 15 SP5 client tools repositories to spacewalk-common-channels
```

Reply:
Well, now you have two entries so you need to remove the one from line 2 :-D

Reply:
Well, but now you left the wrong one ;-)

```suggestion
- Add SLES 15 SP5 client tools repositories to spacewalk-common-channels
```
Just accept this suggestion, and the problem will go away.